src/command/format_stdin.rs

```diff
@@ -21,18 +22,6 @@ pub fn format_stdin_cmd(
         }
     };
 
-    // Search for the treefmt.toml from there.
-    let config_file = match config::lookup(work_dir) {
```

Comment:
Doesn't `config::lookup()` already happen in `cli_from_args()` for both `format_cmd()` and `format_stdin_cmd()`?